src/coincident/search/main.py

```diff
@@ -138,6 +143,34 @@ def search(
 
             gf = stac.to_geopandas(item_collection)
 
+            if dataset.alias == "cop30" or dataset.alias == "worldcover":
+                warnings.warn(
+                    "WARNING: searching large areas for COP30 and/or ESA WorldCover may take a while...",
+                    UserWarning,
+                )
+                # TODO: 1) improve loading speed
+                # tricky when no dates defined for ESA WorldCover as items returns both 2020 and 2021 data
+                # 2) is there a better way of getting search GDF/GeoSeries coordinates?
+                # given aoi = _pystac_client._format_intersects(shapely_geometry)  # to JSON geometry
+                coordinates = aoi["coordinates"][0]
+                total_bounds = (
+                    min(x for x, y in coordinates),
+                    min(y for x, y in coordinates),
+                    max(x for x, y in coordinates),
+                    max(y for x, y in coordinates),
+                )
+                items = stac.to_pystac_items(gf)
+                ds = odc.stac.load(
```

Comment:
This will be slow for large areas because [by default](https://odc-stac.readthedocs.io/en/latest/_api/odc.stac.load.html#odc-stac-load) `chunks=None`. `chunks='auto'` requires dask, but should to be much faster.

Reply:
Ah, great point :blush: I'm assuming we'll eventually add support for dask so I might as well incorporate it here?